content/en/dora_metrics/setup/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Set up DORA Metrics
+kind: documentation
+aliases:
+- /continuous_integration/dora_metrics/setup/
+---
+
+{{< whatsnext desc="Set up DORA Metrics in Datadog:" >}}
```

Comment:
Above the page list here, can we add:

- the request access box
- the information under our current [Overview section](https://docs.datadoghq.com/dora_metrics/?tab=apicurl#overview)
- the blurb under current [Setup section](https://docs.datadoghq.com/dora_metrics/?tab=apicurl#set-up-dora-metrics) - `the Services tracked for the DORA Metrics Deployment Frequency and Change Lead Time must be registered in the [Service Catalog](https://docs.datadoghq.com/tracing/service_catalog). For more information, see [Adding Entries to Service Catalog](https://docs.datadoghq.com/tracing/service_catalog/setup).` ?

Reply:
I have added the request access box and the overview section. For the blurb, I have moved it to the deployments page as it's valid only for deployment events (not for incidents anymore). Should we also have it in this common page in your opinion?

Reply:
Oh right! Let's not put the blurb on this common page. Thanks!